src/Features/Core/Portable/xlf/FeaturesResources.cs.xlf

```diff
@@ -147,6 +157,31 @@
         <target state="new">Remove unused member</target>
         <note />
       </trans-unit>
+      <trans-unit id="Remove_unused_parameter">
+        <source>Remove unused parameter</source>
+        <target state="new">Remove unused parameter</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Remove_unused_parameter_0">
+        <source>Remove unused parameter '{0}'</source>
+        <target state="new">Remove unused parameter '{0}'</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Remove_unused_parameter_0_if_it_is_not_part_of_a_shipped_public_API">
+        <source>Remove unused parameter '{0}' if it is not part of a shipped public API</source>
+        <target state="new">Remove unused parameter '{0}' if it is not part of a shipped public API</target>
```

Comment:
just to check, but these won't be on by default, right?

Reply:
Sounds good.  Given that there's an easy switch to make this "non public" i think that's probably ok...